resources/home/dnanexus/eggd_artemis.py

```
@@ -27,6 +27,8 @@
 @dxpy.entry_point("main")
 def main(
     url_duration,
+    build,
```

Ruff (0.8.0)

Missing type annotation for function argument `build`

(ANN001)